### ISSUE TITLE
help output prints unnecessary number of digits for population parameters

### ISSUE THREAD

**Issue by grahamgower** [popsim-consortium/stdpopsim#603]
**Short description of the problem:**
The help output gives more precision than is useful for mutation rate and recombination rate parameters.

**How to reproduce the problem:**
```
$ stdpopsim HomSap --help
usage: stdpopsim HomSap [-h] [--help-models [HELP_MODELS]] [-b BIBTEX_FILE]
                        [--help-genetic-maps [HELP_GENETIC_MAPS]] [-D] [-g] [-c]
                        [-l LENGTH_MULTIPLIER] [-s SEED] [-d] [-o OUTPUT]
                        samples [samples ...]

Run simulations for Homo sapiens using up-to-date genome information,
genetic maps and simulation models from the literature. NOTE: By
default, the tskit '.trees' binary file is written to stdout,so you
should either redirect this to a file or use the '--output' option to
specify a filename.

Default population parameters for Homo sapiens:
Generation time: 30
Population size: 10000
Mutation rate: 1.2899999999999998e-08
Recombination rate: 1.2051790226570845e-08
...
```

**Suggested fixes:**
The print statements should use, say, 4 digits after the decimal.

**Comment by ragreenburg** [comment]
Mind if I take a stab at this?

**Comment by grahamgower** [comment]
That would be great. Thanks @ragreenburg!

**Comment by grahamgower** [comment]
Closed by #604.
